test/JDBC/expected/latest__verification_cleanup__13_6__sys-all_sql_modules-vu-verify.out

```diff
@@ -0,0 +1,187 @@
+USE sys_all_sql_modules_vu_prepare_db
+GO
+
+-- Test for function
+SELECT
+    definition,
+    uses_ansi_nulls,
+    uses_quoted_identifier,
+    is_schema_bound,
+    uses_database_collation,
+    is_recompiled,
+    null_on_null_input,
+    execute_as_principal_id,
+    uses_native_compilation
+FROM sys.all_sql_modules
+WHERE object_id = OBJECT_ID('sys_all_sql_modules_vu_prepare_f1')
+GO
+~~START~~
+nvarchar#!#bit#!#bit#!#bit#!#bit#!#bit#!#bit#!#int#!#bit
+CREATE OR REPLACE FUNCTION sys_all_sql_modules_vu_prepare_db_dbo.sys_all_sql_modules_vu_prepare_f1()<newline> RETURNS integer<newline> LANGUAGE pltsql<newline>AS '{"version_num": "1", "typmod_array": ["-1"], "original_probin": ""}', $function$BEGIN<newline>    RETURN 1;<newline>END$function$<newline>#!#1#!#1#!#0#!#0#!#0#!#0#!#<NULL>#!#0
+~~END~~
+
+
+-- Test for views
+SELECT
+    definition,
+    uses_ansi_nulls,
+    uses_quoted_identifier,
+    is_schema_bound,
+    uses_database_collation,
+    is_recompiled,
+    null_on_null_input,
+    execute_as_principal_id,
+    uses_native_compilation
+FROM sys.all_sql_modules
+WHERE object_id = OBJECT_ID('sys_all_sql_modules_vu_prepare_v2')
+GO
+~~START~~
+nvarchar#!#bit#!#bit#!#bit#!#bit#!#bit#!#bit#!#int#!#bit
+#!#1#!#1#!#0#!#0#!#0#!#0#!#<NULL>#!#0
+~~END~~
+
+
+-- Test for triggers
+SELECT
+    definition,
+    uses_ansi_nulls,
+    uses_quoted_identifier,
+    is_schema_bound,
+    uses_database_collation,
+    is_recompiled,
+    null_on_null_input,
+    execute_as_principal_id,
+    uses_native_compilation
+FROM sys.sql_modules
+WHERE object_id = (SELECT TOP(1) object_id FROM sys.all_objects WHERE name = 'sys_all_sql_modules_vu_prepare_tr1')
+GO
+~~START~~
+nvarchar#!#bit#!#bit#!#bit#!#bit#!#bit#!#bit#!#int#!#bit
+<NULL>#!#1#!#1#!#0#!#0#!#0#!#0#!#<NULL>#!#0
+~~END~~
+
+
+-- Test for proc
+SELECT
+    definition,
+    uses_ansi_nulls,
+    uses_quoted_identifier,
+    is_schema_bound,
+    uses_database_collation,
+    is_recompiled,
+    null_on_null_input,
+    execute_as_principal_id,
+    uses_native_compilation
+FROM sys.all_sql_modules
+WHERE object_id = OBJECT_ID('sys_all_sql_modules_vu_prepare_p1')
+GO
+~~START~~
+nvarchar#!#bit#!#bit#!#bit#!#bit#!#bit#!#bit#!#int#!#bit
+CREATE OR REPLACE PROCEDURE sys_all_sql_modules_vu_prepare_db_dbo.sys_all_sql_modules_vu_prepare_p1()<newline> LANGUAGE pltsql<newline>AS '{"version_num": "1", "typmod_array": [], "original_probin": ""}', $procedure$SELECT 1$procedure$<newline>#!#1#!#1#!#0#!#0#!#0#!#0#!#<NULL>#!#0
+~~END~~
+
+
+-- Test for system function
```

Comment:
What about C valued system function?

Reply:
Good question, I'll add that to the tests.